posthog-node/src/feature-flags.ts

```diff
@@ -427,10 +419,26 @@ class FeatureFlagsPoller {
         this.featureFlagsByKey = {}
         this.groupTypeMapping = {}
         this.cohorts = {}
-        this.loadedSuccessfullyOnce = false
+        this.loadedSuccessfullyOnce = true
         return
       }
 
+      if (res && res.status === 403) {
```

Comment:
maybe just return early if `res` is undefined/null? we're checking for every condition

Reply:
+1